site/class-1-load/laurens-booij/index.html

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
+<meta charset=utf8>
+<!--Update the title with your username:-->
```

Comment:
Remove this comment (you can do that by clicking on the pencil in the top right of this file here on GitHub).